src/js/containers/covid19/recipient/map/MapContainer.jsx

```diff
@@ -9,20 +9,20 @@ import { connect } from 'react-redux';
 import { isCancel } from 'axios';
 import { uniqueId, keyBy, isEqual } from 'lodash';
 import MapWrapper from 'components/covid19/recipient/map/MapWrapper';
-import AwardFilterButtons from 'components/covid19/recipient/map/AwardFilterButtons';
+import AwardFilterButtons from 'components/covid19/recipient/AwardFilterButtons';
 import MapBroadcaster from 'helpers/mapBroadcaster';
 import LoadingSpinner from 'components/sharedComponents/LoadingSpinner';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import MapMessage from 'components/search/visualizations/geo/MapMessage';
 import RecipientMapTooltip from 'components/covid19/recipient/map/RecipientMapTooltip';
 import {
     centerOfMap,
-    awardTypeFilters,
     filters,
     logMapLayerEvent,
     filtersOnClickHandler,
     tooltipLabels
 } from 'dataMapping/covid19/recipient/map/map';
+import { awardTypeTabs } from 'dataMapping/covid19/covid19';
```

Comment:
This mapping is almost the same as `awardTypeFilters` (removed on line 20 above), but has a property called `internal` instead of `value`.